src/BayesianOptimization.jl

```diff
@@ -111,22 +158,28 @@ function show(io::IO, mime::MIME"text/plain", o::BOpt)
         println(io, "model optimum: $(o.model_optimum)")
         println(io, "model optimizer: $(o.model_optimizer)")
         println(io, "iterations: $(o.iterations.i)/$(o.iterations.N)")
-        println(io, "duration: $(o.duration.now - o.duration.starttime)/$(o.duration.duration) s")
+        println(
+            io,
+            "duration: $(o.duration.now - o.duration.starttime)/$(o.duration.duration) s",
+        )
     end
 end
 
 function initialise_model!(o)
     ys = Float64[]
     xs = Vector{Float64}[]
     for x in o.initializer
-        for j in 1:o.repetitions
+        for j = 1:o.repetitions
```

Comment:
This looks like the JuliaFormatters `DefaultStyle`. I'm not a big fan of the `for j = 1:n` style. Could we maybe use `SciMLStyle`?

Reply:
Yes, of course. I will adjust it.